package.json

```diff
@@ -64,7 +64,7 @@
     "fs-extra": "^11.2.0",
     "jest-axe": "^10.0.0",
     "jest-extended": "^6.0.0",
-    "jsdom": "^27.0.0",
+    "jsdom": "^27.1.0",
```

Comment:
Calling this out: The PR title says `27.0.1`, but this is `27.1.0`. Shouldn't be a problem, but the title is misleading.